services/cosi/encode.go

```diff
@@ -0,0 +1,61 @@
+package cosi
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"github.com/dedis/cothority/lib/crypto"
+	"github.com/dedis/cothority/lib/network"
+	"strings"
+)
+
+// MarshalJSON implements golang's JSON marshal interface
+func (s *ServiceResponse) MarshalJSON() ([]byte, error) {
```

Comment:
Erf. I'm struggling here. On one side, I also think we should stick to one encoding technique. But on the other side, a JSON signature is really really nicer than a protobuf signature. I'm not sure if we should impose stg strict here as long as we did not resolve our issue about defining clear boundaries between External API Services vs Internal messages.